File: JavaScript_Basic_04_2023_SolutionsOfProblems/03.LAB_PB-JS-Nested-Conditional-Statements/P11_FruitShop.js
function fruitShop(input) {
    let fruitType = input[0];
    let weekDay = input[1];
    let quantityFruits = Number(input[2]);
    let fruitCost = 0.0;

    switch (weekDay) {
        case "Monday":
        case "Tuesday":
        case "Wednesday":
        case "Thursday":
        case "Friday":
            switch (fruitType) {
                case "banana":
                    fruitCost = 2.5;
                    break;
                case "apple":
                    fruitCost = 1.2;
                    break;
                case "orange":
                    fruitCost = 0.85;
                    break;
                case "grapefruit":
                    fruitCost = 1.45;
                    break;
                case "kiwi":
                    fruitCost = 2.7;
                    break;
                case "pineapple":
                    fruitCost = 5.5;
                    break;
                case "grapes":
                    fruitCost = 3.85;
                    break;
                default:
                    console.log("error")
                    break;
            }
            break;
        case "Saturday":
        case "Sunday":
            switch (fruitType) {
                case "banana":
                    fruitCost = 2.7;
                    break;
                case "apple":
                    fruitCost = 1.25;
                    break;
                case "orange":
                    fruitCost = 0.9;
                    break;
                case "grapefruit":
                    fruitCost = 1.6;
                    break;
                case "kiwi":
                    fruitCost = 3;
                    break;
                case "pineapple":
                    fruitCost = 5.6;
                    break;
                case "grapes":
                    fruitCost = 4.2;
                    break;
                default:
                    console.log("error")
                    break;
            }
            break;
        default:
            console.log("error")
            break;
    }
    if (fruitCost > 0) {
        console.log((fruitCost * quantityFruits).toFixed(2));
    }
}

fruitShop(["tomato",
    "Monday",
    "0.5"])

File: SoftUni_JAVA_BASIC_Jan_2023_SolutionsOfProblems/P03LABConditionalStatements_Advanced/P11_FruitShop.java
package P03LABConditionalStatements_Advanced;

import java.util.Scanner;

public class P11_FruitShop {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String fruitType = scanner.nextLine();
        String dayOfWeek = scanner.nextLine();
        double quantity = Double.parseDouble(scanner.nextLine());

        if (dayOfWeek.equals("Monday") || dayOfWeek.equals("Tuesday") || dayOfWeek.equals("Wednesday") ||
                dayOfWeek.equals("Thursday") || dayOfWeek.equals("Friday")) {
            if (fruitType.equals("banana")) {
                System.out.printf("%.2f", quantity * 2.5);
            } else if (fruitType.equals("apple")) {
                System.out.printf("%.2f", quantity * 1.2);
            } else if (fruitType.equals("orange")) {
                System.out.printf("%.2f", quantity * 0.85);
            } else if (fruitType.equals("grapefruit")) {
                System.out.printf("%.2f", quantity * 1.45);
            } else if (fruitType.equals("kiwi")) {
                System.out.printf("%.2f", quantity * 2.7);
            } else if (fruitType.equals("pineapple")) {
                System.out.printf("%.2f", quantity * 5.5);
            } else if (fruitType.equals("grapes")) {
                System.out.printf("%.2f", quantity * 3.85);
            } else {
                System.out.println("error");
            }
        } else if (dayOfWeek.equals("Saturday") || dayOfWeek.equals("Sunday")) {
            if (fruitType.equals("banana")) {
                System.out.printf("%.2f", quantity * 2.7);
            } else if (fruitType.equals("apple")) {
                System.out.printf("%.2f", quantity * 1.25);
            } else if (fruitType.equals("orange")) {
                System.out.printf("%.2f", quantity * 0.9);
            } else if (fruitType.equals("grapefruit")) {
                System.out.printf("%.2f", quantity * 1.60);
            } else if (fruitType.equals("kiwi")) {
                System.out.printf("%.2f", quantity * 3);
            } else if (fruitType.equals("pineapple")) {
                System.out.printf("%.2f", quantity * 5.6);
            } else if (fruitType.equals("grapes")) {
                System.out.printf("%.2f", quantity * 4.2);
            } else {
                System.out.println("error");
            }
        } else System.out.println("error");
    }
}
